migrate/repo_checks.py

```diff
@@ -358,33 +443,33 @@ def main(org, dry_run, github_token):
     ]
 
     for repo in repos:
-        if is_security_private_fork(api, org, repo):
-            continue
-
         click.secho(f"{repo}: ")
         for CheckType in CHECKS:
             check = CheckType(api, org, repo)
 
-            result = check.check()
-            if result[0]:
-                color = "green"
-            else:
-                color = "red"
-
-            click.secho(f"\t{result[1]}", fg=color)
-
-            if dry_run:
-                steps = check.dry_run()
-                steps_color = "yellow"
+            if check.is_relevant():
+                result = check.check()
+                if result[0]:
+                    color = "green"
+                else:
+                    color = "red"
+
+                click.secho(f"\t{result[1]}", fg=color)
+
+                if dry_run:
+                    steps = check.dry_run()
+                    steps_color = "yellow"
+                else:
+                    steps = check.fix()
+                    steps_color = "green"
+
+                if steps:
+                    click.secho("\tSteps:\n\t\t", fg=steps_color, nl=False)
+                    click.secho(
+                        "\n\t\t".join([step.replace("\n", "\n\t\t") for step in steps])
```

Comment:
You might like [`textwrap.indent`](https://docs.python.org/3/library/textwrap.html#textwrap.indent).